### DISCUSSION TITLE
Suggested improvements to fill notes "peeking" logic

### DISCUSSION THREAD

**Discussion by seangoodvibes** [SynthstromAudible/DelugeFirmware#599] [Ideas]
### Please describe the problem:

here are some UX improvements that could be made to the fill notes "peeking" logic

1. if a non-fill note was entered on the grid, you then held sync scaling and pressed that non-fill note again, I expected to see that the note would change to fill instead of removing the note and requiring you to re-enter it as fill

2. if you're holding audition and turning select to change the row to fill, I would've expected the the entire row to change to blue while holding audition pad and then switch back to regular colour when you let go of audition

3. same as 1&2: when holding a single note and turning select to change to fill, I expected note colour to change to blue while holding the single note down

### What is the expected behavior?

See above

### Is there a relevant Pull request?

https://github.com/SynthstromAudible/DelugeFirmware/pull/453

### What hardware did you reproduce it with?

OLED

### What firmware did you reproduce it with?

Built from release/1.0

**Comment by trappar** [comment]
+1 on all of the above.

**Comment by ok-reza** [comment]
wanted to share a thought someone had for fill mode:
have a 'mute fill'/'don't fill' option where those notes **won't** play when fill is held.
